baselines/sharepointonedrive.md

```diff
@@ -0,0 +1,339 @@
+GitHub repositories. The respective documents are subject to copyright
+and are adapted under the terms of the Creative Commons Attribution 4.0
+International license. Source documents are linked throughout this
+document. The United States Government has adapted selections of these
+documents to develop innovative and scalable configuration standards to
+strengthen the security of widely used cloud-based software services.
+
+## Assumptions
+
+The **License Requirements** sections of this document assume the
+organization is using an [M365
+E3](https://www.microsoft.com/en-us/microsoft-365/compare-microsoft-365-enterprise-plans)
+or [G3](https://www.microsoft.com/en-us/microsoft-365/government)
+license level. Therefore, only licenses not included in E3/G3 are
+listed.
+
+# Baseline Policies
+
+## 1. External Sharing
+
+This section helps reduce security risks related to the sharing of files with users external to the agency. This includes guest users, users who use a verification code and users who access an Anyone link.
+
+### Policies
+#### MS.SHAREPOINT-ONEDRIVE.1.1v1
+External sharing for Sharepoint SHALL be limited to Existing Guests or Only People in your Organization.
+- _Rationale:_ TODO
+- _Last modified:_ June 2023
+
+#### MS.SHAREPOINT-ONEDRIVE.1.2v1
+External sharing for OneDrive SHALL be limited to Existing Guests or Only People in your Organization.
+- _Rationale:_ TODO
+- _Last modified:_ June 2023
+
+#### MS.SHAREPOINT-ONEDRIVE.1.3v1
+External sharing SHALL be restricted to approved external domains per interagency collaboration needs.
+- _Rationale:_ TODO
+- _Last modified:_ June 2023
+- _Note:_ This policy is only applicable if the external sharing slider on the admin page is set to any value other than Only People in your Organization.
+
+#### MS.SHAREPOINT-ONEDRIVE.1.4v1
+Only users in approved security groups SHALL be allowed to share externally.
+- _Rationale:_ TODO
+- _Last modified:_ June 2023
+- _Note:_ This policy is only applicable if the external sharing slider on the admin page is set to any value other than Only People in your Organization.
+
+#### MS.SHAREPOINT-ONEDRIVE.1.5v1
+Guest access SHALL be limited to the email the invitation was sent to.
+- _Rationale:_ TODO
+- _Last modified:_ June 2023
+- _Note:_ This policy is only applicable if the external sharing slider on the admin page is set to any value other than Only People in your Organization.
+
+### Resources
+
+- [Overview of external sharing in SharePoint and OneDrive in Microsoft 365 \| Microsoft Documents](https://learn.microsoft.com/en-us/sharepoint/external-sharing-overview)
+
+- [Manage sharing settings for SharePoint and OneDrive in Microsoft 365 \| Microsoft Documents](https://docs.microsoft.com/en-us/sharepoint/turn-external-sharing-on-or-off)
+
+### License Requirements
+
+- N/A
+
+### Implementation
+
+All of the settings in this section are configured in the **SharePoint admin center**.
+
+#### MS.SHAREPOINT-ONEDRIVE.1.1v1, instructions:
```

Comment:
Style is inconsistent with new guidelines; remove the "," between the policy id and the word instructions. This comment is applicable to all policy implementation instructions.

Reply:
Fixed it. Thanks.